cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagement.scala

```diff
@@ -124,24 +132,52 @@ object ClusterHttpManagementRoutes extends ClusterHttpManagementHelper {
       }
     }
 
+  private def routeGetShardInfo(cluster: Cluster, shardRegionName: String) =
+    get {
+      extractExecutionContext { implicit executor =>
+        complete {
+          implicit val timeout = Timeout(1.second)
```

Comment:
1 second might be too short, and I don't see a reason for having it too short. `5.seconds`? or is there some general http request timeout that we can use?